docs/luigi-core-api.md

```diff
@@ -496,7 +496,7 @@ Navigates to the given path in the application. It contains either a full absolu
 ```javascript
 Luigi.navigation().navigate('/overview')
 Luigi.navigation().navigate('users/groups/stakeholders')
-Luigi.navigation().navigate('/settings', null, true) // preserve view
+Luigi.navigation().navigate('/settings', null, { title: "Settings Menu", size: "m" }) // openAsModal
```

Comment:
We recommend to use the openAsModal function and not the navigate function.
The `preserve view` example is wrong.
Thanks for finding this 👍 !
Would you change the PR for the preserved view fix
`Luigi.navigation().navigate('/settings', true) // preserve view`

Reply:
Oke @JohannesDoberer  :+1:  thank you